mmdeploy/codebase/mmdet/models/dense_heads/solov2_head.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, List
+
+import torch
+import torch.nn.functional as F
+from mmdet.models.layers.matrix_nms import mask_matrix_nms
+from torch import Tensor
+
+from mmdeploy.codebase.mmdet.deploy import get_post_processing_params
+from mmdeploy.core import FUNCTION_REWRITER
+from mmdeploy.utils import Backend
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmdet.models.dense_heads.solov2_head.'
+    'SOLOV2Head.predict_by_feat',
+    backend=Backend.OPENVINO.value)
```

Comment:
Could you remove backend seting `backend=Backend.OPENVINO.value` and let it use default backend? Cause this rewriting could also be used for other backends like onnxruntime, tensorrt, etc

Reply:
May kindly remove in solo_head as well and rename function name to `solohead__predict_by_feat`
https://github.com/open-mmlab/mmdeploy/blob/6c26e887d4300c3afb57a0900c6ad53798d9dfda/mmdeploy/codebase/mmdet/models/dense_heads/solo_head.py#L15